test/e2e/run_staticip_test.go

```diff
@@ -105,6 +105,11 @@ var _ = Describe("Podman run with --ip flag", func() {
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
 
+		// We need to set "no_proxy" in proxy environment
+		if env, found := os.LookupEnv("no_proxy"); found {
+			os.Setenv("no_proxy", ip)
+			defer os.Setenv("no_proxy", env)
+		}
```

Comment:
Does this now still break in a proxy env since you only set no_proxy when it was already set on the host?
What I meant was lookup env, if not found set env for the test and unset env afterwards,
if found set env for the test and set env back to the same value as before.